the-list-ui/src/App.js

```diff
@@ -1,9 +1,27 @@
+import { useEffect, useState } from 'react';
 import './App.css';
 import Input from './components/Input';
+import { getStudents } from './request-functions';
 
 function App() {
+  const [students, setStudents] = useState([{ _id: '', name: '' }]);
```

Comment:
Зачем добавлять в массив пустую запись?

```suggestion
  const [students, setStudents] = useState([]);
```

Reply:
Чтобы вс код не ругался на students._id и students.name

Reply:
@OstaptsovDanil, и для этого ты создаёшь пустого студента, который приведёт к рендеру пустой строчки (если данные с сервера будут грузиться долго)?

Надо иначе тип задать для этих переменных.

Reply:
Что конкретно у тебе пишет, когда ты убираешь этого несуществующего студента? Мы используем JS, не TS, он не должен настолько жёстко проверять.

<img width="440" alt="image" src="https://user-images.githubusercontent.com/4187729/198952336-fb33cda6-529d-40ee-8e57-9cae4a988d67.png">

Reply:
У меня не расстраивается.